src/main/java/bio/terra/service/snapshotbuilder/utils/CriteriaQueryBuilder.java

```diff
@@ -97,35 +98,32 @@ FilterVariable generateFilter(SnapshotBuilderDomainCriteria domainCriteria) {
     TablePointer occurrencePointer = TablePointer.fromTableName(domainOption.getTableName());
     TableVariable occurrenceVariable = TableVariable.forPrimary(occurrencePointer);
 
-    TablePointer ancestorPointer = TablePointer.fromTableName("concept_ancestor");
+    TablePointer ancestorPointer =
+        TablePointer.fromTableName(ConceptAncestorConstants.CONCEPT_ANCESTOR);
     TableVariable ancestorVariable =
         TableVariable.forJoined(
             ancestorPointer,
-            "ancestor_concept_id",
+            ConceptAncestorConstants.DESCENDANT_CONCEPT_ID,
             new FieldVariable(
                 new FieldPointer(occurrencePointer, domainOption.getColumnName()),
                 occurrenceVariable));
 
     return new SubQueryFilterVariable(
-        getFieldVariableForRootTable(PERSON_ID_FIELD_NAME),
+        getFieldVariableForRootTable(PersonConstants.PERSON_ID),
         SubQueryFilterVariable.Operator.IN,
```

Comment:
While changing this, you could also update it to use `SubQueryFilterVariable.in()`, `BooleanAndOrFilterVariable.or()`, `BinaryFilterVariable.equals()`.